verification/cpp/suite/test_constant.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2024 OpenCyphal Development Team.
+ * Authors: Sergei Shirokov <sergei.shirokov@zubax.com>
+ * This software is distributed under the terms of the MIT License.
+ *
+ * Tests of constant
+ */
+
+#include "gmock/gmock.h"
+#include "regulated/basics/Struct__0_1.hpp"
+#include "regulated/basics/Union_0_1.hpp"
+#include "regulated/basics/Service_0_1.hpp"
+
+using testing::Le;
+using testing::StrEq;
+using testing::FloatNear;
+using testing::DoubleNear;
+
+TEST(ConstantTests, Struct)
+{
+    // Type parameters.
+    EXPECT_TRUE(regulated::basics::Struct__0_1::_traits_::HasFixedPortID);
+    EXPECT_THAT(regulated::basics::Struct__0_1::_traits_::FixedPortId, 7000);
+    EXPECT_THAT(regulated::basics::Struct__0_1::_traits_::FullNameAndVersion(), StrEq("regulated.basics.Struct_.0.1"));
+
+    // Application constants.
+    EXPECT_THAT(regulated::basics::Struct__0_1::CONSTANT_MINUS_THREE, DoubleNear(-3.0, 1e-9));
+    EXPECT_THAT(regulated::basics::Struct__0_1::CONSTANT_ZEE, 'Z');
+    EXPECT_THAT(-regulated::basics::Struct__0_1::CONSTANT_MINUS_MAX_OFFSET,
+                Le(regulated::basics::Struct__0_1::_traits_::SerializationBufferSizeBytes * 8U));
+    EXPECT_TRUE(regulated::basics::Struct__0_1::CONSTANT_TRUTH);
+
+// TODO: Uncomment when some form of "ARRAY_CAPACITY" is generated.
```

Comment:
Separate issue and PR but probably I will add generation of a new `<DSDL Type>::_traits_::ArrayCapacity` struct,
where there will be listed `static constexpr std::size_t` constants for each field which has variable array type. Static array fields don't require such information (b/c `std::array<>` already has `.size()` method), but maybe just for consistency I will list them as well.

Reply:
@thirtytwobits Do you agree with my future plan about above "ARRAY_CAPACITY" topic?

Reply:
CETL's variable_length_array already provides max_size for this?

Reply:
Yes it does, but under c++17 and above we use `std::vector`